experimental/bulk-txn-submit/src/main.rs

```diff
@@ -0,0 +1,133 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+use anyhow::{Context, Result};
+use aptos_experimental_bulk_txn_submit::{
+    coordinator::{
+        create_sample_addresses, execute_return_worker_funds, execute_submit, CleanAddresses,
+        CreateSampleAddresses, SubmitArgs,
+    },
+    workloads::{
+        create_account_addresses_work, CreateAndTransferAptSignedTransactionBuilder,
+        TransferAptSignedTransactionBuilder,
+    },
+};
+use aptos_logger::{Level, Logger};
+use aptos_sdk::move_types::account_address::AccountAddress;
+use aptos_transaction_emitter_lib::Cluster;
+use clap::{Parser, Subcommand};
+use rand::{seq::SliceRandom, thread_rng};
+use std::collections::HashSet;
+
+#[derive(Parser, Debug)]
+struct Args {
+    #[clap(subcommand)]
+    command: DemoCommand,
+}
+
+#[derive(Subcommand, Debug)]
+enum DemoCommand {
+    Submit(Submit),
+    CreateSampleAddresses(CreateSampleAddresses),
+    CleanAddresses(CleanAddresses),
```

Comment:
Perhaps `FilterAddresses` or `SanitizeAddresses`? From the command name, I was imagining this somehow "removed" addresses, such as removing balances from all addresses.